utils/releaser/config/services.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+use GuzzleHttp\Client;
+use GuzzleHttp\ClientInterface;
+use Shopsys\Releaser\Command\ReleaseCommand as ShopsysReleaseCommand;
+use Shopsys\Releaser\FileManipulator\MonorepoUpgradeFileManipulator;
+use Shopsys\Releaser\FilesProvider\ComposerJsonFilesProvider;
+use Shopsys\Releaser\ReleaseWorker\ReleaseWorkerProvider as ShopsysReleaseWorkerProvider;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symplify\MonorepoBuilder\Release\Command\ReleaseCommand;
+use Symplify\MonorepoBuilder\Release\ReleaseWorkerProvider;
+use Twig\Environment;
+use Twig\Loader\FilesystemLoader;
+use Twig\Loader\LoaderInterface;
+
+/**
+ * @param \Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator $containerConfigurator
+ */
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $parameters = $containerConfigurator->parameters();
+    $services = $containerConfigurator->services();
+
+    # name of the monorepo package, for UPGRADE.md and CHANGELOG.md links
+    $parameters->set('monorepo_package_name', 'shopsys/shopsys');
+
+    # these stages can use already existing tag; newer version than the last one is required in all other stages by default
+    # this can be used to skip the tag validation completely
+    # we need this as we are supporting multiple versions and we need to release tags that are lower than the newest one
+    # e.g. we want to release "v7.3.2" even if "v8.0.0" is already released
+
+    $parameters->set('stages_to_allow_existing_tag', ['release-candidate', 'release', 'after-release']);
+
+    $containerConfigurator->import(__DIR__ . '/release-candidate.php');
+    $containerConfigurator->import(__DIR__ . '/release.php');
+    $containerConfigurator->import(__DIR__ . '/after-release.php');
+
+    $services->defaults()
+        ->autowire()
+        ->autoconfigure();
+
+    $services->load('Shopsys\Releaser\\', __DIR__ . '/../src')
```

Comment:
I truly did try it :) it was helpful and saved me a lot of time. 

Well, I have no strong feelings one way or the other... With plugins in PhpStorm I had no quarrel with yaml files. 
But the mentioned article sums up the advantages. 

But back to the original question – two backslashes at the end are mandatory, otherwise, it escapes the single quote resulting in a syntax error. 
The first backslash does not matter if it's one or two – see https://3v4l.org/lhWJU.
But for the sake of unifying, I replaced it with two backslashes in 59bef9f25

Reply:
I see, thanks for the answer